src/model/Game.java

```diff
@@ -220,6 +245,32 @@ public void startDay() {
         resetDay(); // not sure if we want to change resetDay so it will call startDay again if all is well
     }
 
+    // TODO: need to figure out how to save the clearing the player wants to move to
+    // and the weapons/armour (??) they want to alert/unalert
+    public void doTodaysActivities(Player player) {
+    	for (int i = 0; i < player.activities.length; i++) {
+    		// check if they can block another player
+    		Player[] players = block(player);
+    		for (int j = 0; j < players.length; j++) {
+    			if (players[j] != null) {
+    				// TODO: the player has the option to block them. Should I just do that automatically for now??
```

Comment:
@SunsetBlade I wasn't sure networking wise if I should for now just automatically have the player block any unhidden players in the clearing or if you'd be able to send them the option of players that  they can block